### DISCUSSION TITLE
DC7F1559-4B6F-C164-CFC1-4AE9701C322A

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#76] [General]
# 82 Eridani (Unconfirmed Planets)

This addon depicts the four additional planets of the star 82 Eridani whose existence is yet to be confirmed by further analyses. If you're looking for this star's confirmed planets, you can download them from the 

https://celestia.mobi/addon?item=DC7F1559-4B6F-C164-CFC1-4AE9701C322A&lang=en

**Comment by exocritic5457** [comment]
Ok so the alternate surfaces have a problem. It seems like the planets' names were switched at some point because f (really g before it was changed?) has g's texture, e has f's, g has c's, and d has e's. Heard c was disproven, maybe that could be yet another addon, or maybe not. I'll let someone else decide on that. 

Also strange in general how this outer gas giant has had no mentions anywhere else.
